utility/copy_file.py

```diff
@@ -0,0 +1,40 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CopyFile:
+    @staticmethod
+    def run(kw, args):
+        """Copy the file from one instance to another.
+
+        Copy the specified file from source to destination instance.
+
+        Usage
+        - copy_file: True
+          source: ceph-rbd1
+          dest: ceph-rbd2
+          path: ./dummy_file
+        """
+
+        cluster = kw["ceph_cluster_dict"][args["source"]]
+
+        for node in cluster:
```

Comment:
IMO, we can have this in an utility

Reply:
changed it to calling an existing method from ceph object (will finalize after testing)